monkey/monkey_island/cc/ui/src/styles/pages/RunServerPage.scss

```diff
@@ -1,3 +1,11 @@
+h1.page-title{
+  margin-bottom: 0px;
+}
+
```

Comment:
@VakarisZ This gets applied to all pages. Is that what we want? If so, it should go in App.css or Main.scss (I'm not sure what the difference between them is.)

Reply:
This is def. not what we wanted. Let me change it.

Reply:
Yes, this is a bug. This gets applied to all `h1.page-title` elements. We need to add a class and have a specific selector, like `.getting-started-page.h1.page-title`